File: backend/src/main/java/com/backend/domain/repository/repository/RepositoryLanguageRepository.java
package com.backend.domain.repository.repository;

import com.backend.domain.repository.entity.RepositoryLanguage;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.List;

public interface RepositoryLanguageRepository extends JpaRepository<RepositoryLanguage, Long> {
    List<RepositoryLanguage> findByRepositories_Id(Long repositoriesId);
}

File: backend/src/main/java/com/backend/domain/repository/repository/RepositoryLanguageRepository.kt
package com.backend.domain.repository.repository

import com.backend.domain.repository.entity.RepositoryLanguage
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface RepositoryLanguageRepository : JpaRepository<RepositoryLanguage, Long> {
    fun findByRepositories_Id(repositoriesId: Long): List<RepositoryLanguage>
}